server/app.js

```diff
@@ -4,14 +4,15 @@ const path = require("path");
 const autoLoad = require("fastify-autoload");
 
 const app = fastify({ debug: false });
+const port = require("../port.json").port;
```

Comment:
Dibikin dia ada default portnya ke `3000`

```suggestion
const port = process.env.PORT || 3000;
```